bin/sl-pm.js

```diff
@@ -169,3 +202,12 @@ function stopWhenDone(/* $0, app */) {
   });
   */
 }
+
+function checkAndUpgradeDb(baseDir, callback) {
```

Comment:
No this checks if an upgrade is needed, the one in upgrade-db.js performs the upgrade and throws error if something goes wrong.